components/feature/qr/src/main/java/mozilla/components/feature/qr/QrFeature.kt

```diff
@@ -34,7 +35,9 @@ class QrFeature(
     private val context: Context,
     private val fragmentManager: FragmentManager,
     private val onScanResult: OnScanResult = { },
-    override val onNeedToRequestPermissions: OnNeedToRequestPermissions = { }
+    override val onNeedToRequestPermissions: OnNeedToRequestPermissions = { },
+    @StringRes
+    private var scanMessage: Int? = null
```

Comment:
nit: Let's add some docs for this new property :)

Reply:
@Amejia481 Comments in code or entry in changelog?

Reply:
both 😄

Reply:
Initially I was thinking about the code but now that you mentioned the changelog let's add it too XD

Reply:
Done! Let me know if you have any feedback on content.

Reply:
Looks good!